src/context/SelectedMassetProvider.ts

```diff
@@ -5,7 +5,7 @@ import { MassetName } from '../types';
 
 const [useSelectedMassetCtx, SelectedMassetProvider] = createStateContext<
   MassetName
->('mUSD');
+>('mBTC');
```

Comment:
Changed because of the save contract deployed to Ropsten, obviously will be changed later